### ISSUE TITLE
NVDA fails to read typed characters on Windows 10

### ISSUE THREAD

**Issue by supanut2000** [nvaccess/nvda#5611]
NVDA fails to read typed characters on Windows Store apps on Windows 10, E.G the Store, Facebook and even the Windows search.

**Comment by jcsteh** [comment]
Is this an installed copy or a portable copy? Is it an official build of 
NVDA (not provided by someone other than NV Access)?

Portable copies don't have the necessary security privileges to access 
typed characters in Windows Store-style apps, which now includes much of 
the Windows shell itself such as the Start Menu. The same is true for 
unofficial (technically unsigned) builds. There's nothing we can do 
about this; it is security enforced by Windows.

**Comment by supanut2000** [comment]
I'm using an official, stable 2015.4 build, and I'm using the install copy of NVDA. I've tested this in a virtual machine running Windows 10 and the same issue occurs. Sorry if I didn't specify the milestone when creating the issue earlier...

**Comment by jcsteh** [comment]
Okay. This seems to work for me on Windows 10 with an installed copy.

What exact build of Windows 10 are you using? You can get this by 
pressing Windows+r to open the Run dialog, typing "winver" (without the 
quotes) and pressing enter.

**Comment by supanut2000** [comment]
I'm using the 10586 build, the November update. I did not install the RTM build earlier; I performed a direct upgrade from Windows 7 to Windows 10 November update.

**Comment by supanut2000** [comment]
Update I just tried the latest snapshot, and it seems that this issue has been fixed.

**Comment by ehollig** [comment]
It sounds like, from https://github.com/nvaccess/nvda/issues/5611#issuecomment-165316397 this issue has been resolved. I also was able to have NVDA speak typed characters in Edge and the Store with addons disabled.

**Comment by josephsl** [comment]
Hi, resolved in Creators Update and nVDA 2017.2 Will close. Thanks.

**Comment by jcsteh** [comment]
Reporting of typed characters in UWP apps was fixed in #6017. Closing as a duplicate of #6017.
